api/src/test/scala/hmda/api/http/institutions/submissions/SubmissionEditPathsSpec.scala

```diff
@@ -94,9 +81,14 @@ class SubmissionEditPathsSpec extends InstitutionHttpApiSpec with LarGenerators
 
       // Check for correct baseline validation state.
       //   Incorrect baseline will invalidate the other tests in this section.
-      val state: HmdaFileValidationState = Await.result(fValidationState, 5.seconds)
-      state.validityErrors.isEmpty mustBe true
-      state.syntacticalErrors.isEmpty mustBe true
+      val fState = for {
+        s <- fHmdaValidatorActor(2)
+        xs <- (s ? GetState).mapTo[HmdaVerificationState]
+        svState <- (s ? GetSVState).mapTo[SVState]
+      } yield (xs, svState.containsSVEdits)
+      val (state: HmdaVerificationState, containsSVEdits: Boolean) = Await.result(fState, 5.seconds)
+
+      containsSVEdits mustBe false
       state.macroVerified mustBe false
       state.qualityVerified mustBe false
```

Comment:
You can get rid of the blocking `Await.result` by extending `InstitutionHttpApiAsyncSpec` instead of `InstitutionHttpApiSpec` and replacing the previous 4 lines with:

```scala
fState.map {
        case (state: HmdaVerificationState, containsSVEdits: Boolean) =>
          containsSVEdits mustBe false
          state.macroVerified mustBe false
          state.qualityVerified mustBe false
      }
```

Reply:
Good call. Done in 67ef1ebc8b75c7128696d4c1f08138479d75095e.